From 6eb969056aa6a02f5e60e878a45c4b55ff7926ad Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@toradex.com>
Date: Mon, 14 Sep 2020 10:41:29 +0200
Subject: [PATCH] colibri-imx6ull: use distro boot by default

Use distro boot for raw NAND modules by default. This is required
for TorizonCore.

Also set bootubivol to rootfs.

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
Signed-off-by: Sergio Prado <sergio.prado@toradex.com>
Signed-off-by: Ming Liu <ming.liu@toradex.com>
---
 include/configs/colibri-imx6ull.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/include/configs/colibri-imx6ull.h b/include/configs/colibri-imx6ull.h
index 7bce86b219..c2d680fda7 100644
--- a/include/configs/colibri-imx6ull.h
+++ b/include/configs/colibri-imx6ull.h
@@ -60,10 +60,11 @@
 		"ubi read ${fdt_addr_r} dtb && " \
 		"run fdt_fixup && bootz ${kernel_addr_r} - ${fdt_addr_r}\0" \
 
-/* Run Distro Boot script if ubiboot fails */
-#define CONFIG_BOOTCOMMAND "run ubiboot || run distro_bootcmd;"
+/* Run Distro Boot Script */
+#define CONFIG_BOOTCOMMAND "run distro_bootcmd;"
 
 #define BOOT_TARGET_DEVICES(func) \
+	func(UBIFS, ubifs, 0) \
 	func(MMC, mmc, 0) \
 	func(USB, usb, 0) \
 	func(DHCP, dhcp, na)
@@ -78,6 +79,7 @@
 	UBOOT_UPDATE \
 	"boot_script_dhcp=boot.scr\0" \
 	"bootubipart=ubi\0" \
+	"bootubivol=rootfs\0" \
 	"console=ttymxc0\0" \
 	"defargs=user_debug=30\0" \
 	"dfu_alt_info=" DFU_ALT_NAND_INFO "\0" \
-- 
2.29.0

